content/docs/rfcs/66/README.md

```diff
@@ -0,0 +1,32 @@
+---
+slug: 66
+title: 66/WAKU2-METADATA
+name: Waku Metadata Protocol
+status: raw
+editor: Alvaro Revuelta <alrevuelta@status.im>
+contributors:
+---
+
+# Metadata Protocol
+
+Waku specifies a req/resp protocol that provides information about the node's medatadata.
+
+
+## Protocol id
+
+`/vac/waku/metadata/1.0.0`
+
+## Request
+
+```proto
+message WakuMetadataRequest {
+}
+```
+
+## Response
+
+```proto
+message WakuMessageResponse {
```

Comment:
```suggestion
message WakuMetadataResponse {
```

Not sure if typo or if there is some other intention that I'm missing.

Reply:
fixed d381ee1bbdaf033a4561eda773048117e2be731d